iquip/apps/builder.py

```diff
@@ -653,7 +747,9 @@ def submit(self):
         """
         try:
             experimentArgs = self.argumentsFromListWidget(self.builderFrame.argsListWidget)
+            scanArgs = self.scannableFromListWidget(self.builderFrame.scanListWidget)
             schedOpts = self.argumentsFromListWidget(self.builderFrame.schedOptsListWidget)
+            experimentArgs.update(scanArgs)
```

Comment:
A `update()` never causes a `ValueError`, thus please move it out of the `try-except` statement.

Reply:
Please check this.